pkgs/by-name/pa/passes/package.nix

```
@@ -42,23 +44,24 @@ stdenv.mkDerivation (finalAttrs: {
     meson
     ninja
     pkg-config
-    (python3.withPackages (pp: [ pp.pygobject3 ]))
+    pythonEnv
```

```suggestion
    python3
```
is not enough at build time?